default-recommendations/conditional-suggestions-test.rkt

```diff
@@ -0,0 +1,33 @@
+#lang resyntax/testing/refactoring-test
+
+
+require: resyntax/default-recommendations conditional-suggestions
+
+
+test: "if-else *doesn't* refactor to cond"
+------------------------------
+#lang racket/base
+(if 'cond 'then 'else)
+------------------------------
+
+
+test: "singly-nested if-else *doesn't* refactor to cond"
```

Comment:
Refactoring this case seems useful. Is there a particular reason you wanted to avoid touching it?